tests/test_tasker.py

```diff
@@ -369,3 +374,33 @@ def version(self):
         .once())
 
     DockerTasker()
+
+
+def my_func(*args, **kwargs):
+    my_args = ('some', 'new')
+    my_kwargs = {'one': 'first', 'two': 'second'}
+    assert args == my_args
+    assert kwargs == my_kwargs
+    response = requests.Response()
+    response.status_code = 408
+    raise APIError("test fail", response)
+
+
+@pytest.mark.parametrize('retry_times', [-1, 0, 1, 2, 3])
```

Comment:
Why is `retry_times==-1` being tested here? Retries are not part of public API, it cannot be negative (and a string btw) outside of atomic-reactor

Reply:
that is testing that function isn't run at all, when retry_times = 0, func will run 1 time, same logic as with session